commands/user.go

```diff
@@ -352,6 +364,49 @@ func resetUserMfaCmdF(c client.Client, cmd *cobra.Command, args []string) error
 	return nil
 }
 
+func deleteUser(c client.Client, user *model.User) (bool, *model.Response) {
+	return c.PermanentDeleteUser(user.Id)
+}
+
+func getUserDeleteConfirmation() error {
+	var confirm string
+	fmt.Println("Have you performed a database backup? (YES/NO): ")
+	fmt.Scanln(&confirm)
+
+	if confirm != "YES" {
+		return errors.New("aborted: You did not answer YES exactly, in all capitals")
+	}
+	fmt.Println("Are you sure you want to delete the users specified? All data will be permanently deleted? (YES/NO): ")
+	fmt.Scanln(&confirm)
+	if confirm != "YES" {
+		return errors.New("aborted: You did not answer YES exactly, in all capitals")
+	}
+	return nil
+}
+
+func deleteUsersCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	confirmFlag, _ := cmd.Flags().GetBool("confirm")
+	if !confirmFlag {
+		if err := getUserDeleteConfirmation(); err != nil {
+			return err
+		}
```

Comment:
I did this because `dupl` complains that this function is too similar to  `deleteTeamsCmdF`